### PR TITLE
[JBPM-9837] Adding indexes in the case log tables

### PR DESCRIPTION
**Pull request by elguardian**
Jira: https://issues.redhat.com/browse/JBPM-9837

### CODE REVIEW

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/oracle/jbpm-7.58-to-7.59.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+create unique index IDX_CorrelationKeyInfo_name on CorrelationKeyInfo(name);
```
Comment (gmunozfe) [inline jbpm-db-scripts/src/main/resources/db/upgrade-scripts/oracle/jbpm-7.58-to-7.59.sql:1]: This is already part of the springboot schema
https://github.com/kiegroup/jbpm/blob/940bcf5698db5d95a911291f2c2cf4b1930fac4a/jbpm-db-scripts/src/main/resources/db/ddl-scripts/oracle/oracle-springboot-jbpm-schema.sql#L101

Are we sharing the same upgrade script for both?

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/postgresql/jbpm-7.58-to-7.59.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
+create unique index IDX_CorrelationKeyInfo_name on CorrelationKeyInfo(name);
```
Comment (gmunozfe) [inline jbpm-db-scripts/src/main/resources/db/upgrade-scripts/postgresql/jbpm-7.58-to-7.59.sql:1]: Same question for postgresql.

**Review by gmunozfe — COMMENTED** [review]
Looks good to me, well done @elguardian !
Just a couple of questions regarding the inclusion of `IDX_CorrelationKeyInfo_name` in upgrade scripts for oracle and postgresql (those named as **springboot** already contain it)

**Comment by gmunozfe** [comment]
jenkins retest this

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/oracle/jbpm-7.58-to-7.59.sql (hunk 1 above)
Comment (elguardian) [reply jbpm-db-scripts/src/main/resources/db/upgrade-scripts/oracle/jbpm-7.58-to-7.59.sql:1]: I was suprised by that, but only happens with the autogenerated means (bytea and new spring boot scripts). We cannot do much about it but add some release notes

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/postgresql/jbpm-7.58-to-7.59.sql (hunk 2 above)
Comment (elguardian) [reply jbpm-db-scripts/src/main/resources/db/upgrade-scripts/postgresql/jbpm-7.58-to-7.59.sql:1]: I was suprised by that, but only happens with the autogenerated means (bytea and new spring boot scripts). We cannot do much about it but add some release notes

File: jbpm-db-scripts/src/main/resources/db/ddl-scripts/sqlserver/sqlserver-jbpm-schema.sql [hunk 3]
```diff
@@ -828,4 +828,12 @@
 
     create index IDX_TaskVariableImpl_taskId on TaskVariableImpl(taskId);
     create index IDX_TaskVariableImpl_pInstId on TaskVariableImpl(processInstanceId);
-    create index IDX_TaskVariableImpl_processId on TaskVariableImpl(processId);
\ No newline at end of file
+    create index IDX_TaskVariableImpl_processId on TaskVariableImpl(processId);
+
+    create unique index IDX_CorrelationKeyInfo_name on CorrelationKeyInfo(name);
```
Comment (gmunozfe) [inline jbpm-db-scripts/src/main/resources/db/ddl-scripts/sqlserver/sqlserver-jbpm-schema.sql:833]: Same here

File: jbpm-db-scripts/src/main/resources/db/ddl-scripts/sqlserver2008/sqlserver2008-jbpm-schema.sql [hunk 4]
```diff
@@ -828,4 +828,12 @@
 
     create index IDX_TaskVariableImpl_taskId on TaskVariableImpl(taskId);
     create index IDX_TaskVariableImpl_pInstId on TaskVariableImpl(processInstanceId);
-    create index IDX_TaskVariableImpl_processId on TaskVariableImpl(processId);
\ No newline at end of file
+    create index IDX_TaskVariableImpl_processId on TaskVariableImpl(processId);
+    
+    create unique index IDX_CorrelationKeyInfo_name on CorrelationKeyInfo(name);
```
Comment (gmunozfe) [inline jbpm-db-scripts/src/main/resources/db/ddl-scripts/sqlserver2008/sqlserver2008-jbpm-schema.sql:833]: Same here

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/postgresql/jbpm-7.58-to-7.59.sql [hunk 5]
```diff
@@ -0,0 +1,5 @@
+create unique index if not exists IDX_CorrelationKeyInfo_name on CorrelationKeyInfo(name);
```
Comment (gmunozfe) [inline jbpm-db-scripts/src/main/resources/db/upgrade-scripts/postgresql/jbpm-7.58-to-7.59.sql:1]: Not needed, as already exists

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/postgresql/rhpam-7.11-to-7.12.sql [hunk 6]
```diff
@@ -4,5 +4,10 @@ alter table TaskEvent add column end_date timestamp;
 alter table NodeInstanceLog add column end_date timestamp;
 alter table VariableInstanceLog add column end_date timestamp;
 create index IDX_TaskEvent_processInstanceId on TaskEvent (processInstanceId);
+create unique index if not exists IDX_CorrelationKeyInfo_name on CorrelationKeyInfo(name);
```
Comment (gmunozfe) [inline jbpm-db-scripts/src/main/resources/db/upgrade-scripts/postgresql/rhpam-7.11-to-7.12.sql:7]: Same here

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/sqlserver/jbpm-7.58-to-7.59.sql [hunk 7]
```diff
@@ -0,0 +1,5 @@
+create unique index IDX_CorrelationKeyInfo_name on CorrelationKeyInfo(name);
```
Comment (gmunozfe) [inline jbpm-db-scripts/src/main/resources/db/upgrade-scripts/sqlserver/jbpm-7.58-to-7.59.sql:1]: Same

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/sqlserver/rhpam-7.11-to-7.12.sql [hunk 8]
```diff
@@ -4,6 +4,11 @@ alter table TaskEvent add end_date datetime;
 alter table NodeInstanceLog add end_date datetime;
 alter table VariableInstanceLog add end_date datetime;
 create index IDX_TaskEvent_processInstanceId on TaskEvent (processInstanceId);
+create unique index IDX_CorrelationKeyInfo_name on CorrelationKeyInfo(name);
```
Comment (gmunozfe) [inline jbpm-db-scripts/src/main/resources/db/upgrade-scripts/sqlserver/rhpam-7.11-to-7.12.sql:7]: Same

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/sqlserver2008/jbpm-7.58-to-7.59.sql [hunk 9]
```diff
@@ -0,0 +1,5 @@
+create unique index IDX_CorrelationKeyInfo_name on CorrelationKeyInfo(name);
```
Comment (gmunozfe) [inline jbpm-db-scripts/src/main/resources/db/upgrade-scripts/sqlserver2008/jbpm-7.58-to-7.59.sql:1]: Same

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/sqlserver2008/rhpam-7.11-to-7.12.sql [hunk 10]
```diff
@@ -4,5 +4,10 @@ alter table TaskEvent add end_date datetime;
 alter table NodeInstanceLog add end_date datetime;
 alter table VariableInstanceLog add end_date datetime;
 create index IDX_TaskEvent_processInstanceId on TaskEvent (processInstanceId);
+create unique index IDX_CorrelationKeyInfo_name on CorrelationKeyInfo(name);
```
Comment (gmunozfe) [inline jbpm-db-scripts/src/main/resources/db/upgrade-scripts/sqlserver2008/rhpam-7.11-to-7.12.sql:7]: Same

File: jbpm-db-scripts/src/main/resources/db/ddl-scripts/mysql5/mysql5-jbpm-schema.sql [hunk 11]
```diff
@@ -848,3 +848,13 @@
     create index IDX_TaskVariableImpl_processId on TaskVariableImpl(processId);
 
     create index IDX_PAsPot_TaskEntity ON PeopleAssignments_PotOwners (task_id, entity_id);
+
+    create unique index IDX_CorrelationKeyInfo_name on CorrelationKeyInfo(name);
```
Comment (gmunozfe) [inline jbpm-db-scripts/src/main/resources/db/ddl-scripts/mysql5/mysql5-jbpm-schema.sql:852]: Same

File: jbpm-db-scripts/src/main/resources/db/ddl-scripts/mysqlinnodb/mysql-innodb-jbpm-schema.sql [hunk 12]
```diff
@@ -851,3 +851,14 @@
     create index IDX_TaskVariableImpl_processId on TaskVariableImpl(processId);
 
     create index IDX_PAsPot_TaskEntity ON PeopleAssignments_PotOwners (task_id, entity_id);
+
+    create unique index IDX_CorrelationKeyInfo_name on CorrelationKeyInfo(name);
```
Comment (gmunozfe) [inline jbpm-db-scripts/src/main/resources/db/ddl-scripts/mysqlinnodb/mysql-innodb-jbpm-schema.sql:855]: Same

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/mysql5/jbpm-7.58-to-7.59.sql [hunk 13]
```diff
@@ -0,0 +1,5 @@
+create unique index IDX_CorrelationKeyInfo_name on CorrelationKeyInfo(name);
```
Comment (gmunozfe) [inline jbpm-db-scripts/src/main/resources/db/upgrade-scripts/mysql5/jbpm-7.58-to-7.59.sql:1]: Same

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/mysql5/rhpam-7.11-to-7.12.sql [hunk 14]
```diff
@@ -4,3 +4,8 @@ alter table TaskEvent add column end_date datetime;
 alter table NodeInstanceLog add column end_date datetime;
 alter table VariableInstanceLog add column end_date datetime;
 create index IDX_TaskEvent_processInstanceId on TaskEvent (processInstanceId);
+create unique index IDX_CorrelationKeyInfo_name on CorrelationKeyInfo(name);
```
Comment (gmunozfe) [inline jbpm-db-scripts/src/main/resources/db/upgrade-scripts/mysql5/rhpam-7.11-to-7.12.sql:7]: Same

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/mysqlinnodb/jbpm-7.58-to-7.59.sql [hunk 15]
```diff
@@ -0,0 +1,5 @@
+create unique index IDX_CorrelationKeyInfo_name on CorrelationKeyInfo(name);
```
Comment (gmunozfe) [inline jbpm-db-scripts/src/main/resources/db/upgrade-scripts/mysqlinnodb/jbpm-7.58-to-7.59.sql:1]: Same

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/mysqlinnodb/rhpam-7.11-to-7.12.sql [hunk 16]
```diff
@@ -4,3 +4,8 @@ alter table TaskEvent add column end_date datetime;
 alter table NodeInstanceLog add column end_date datetime;
 alter table VariableInstanceLog add column end_date datetime;
 create index IDX_TaskEvent_processInstanceId on TaskEvent (processInstanceId);
+create unique index IDX_CorrelationKeyInfo_name on CorrelationKeyInfo(name);
```
Comment (gmunozfe) [inline jbpm-db-scripts/src/main/resources/db/upgrade-scripts/mysqlinnodb/rhpam-7.11-to-7.12.sql:7]: Same

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/oracle/jbpm-7.58-to-7.59-CorrelationKeyInfo_name.sql [hunk 17]
```diff
@@ -0,0 +1 @@
+create unique index IDX_CorrelationKeyInfo_name on CorrelationKeyInfo(name);
```
Comment (gmunozfe) [inline jbpm-db-scripts/src/main/resources/db/upgrade-scripts/oracle/jbpm-7.58-to-7.59-CorrelationKeyInfo_name.sql:1]: Same

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/oracle/rhpam-7.11-to-7.12-CorrelationKeyInfo_name.sql [hunk 18]
```diff
@@ -0,0 +1,2 @@
+create unique index IDX_CorrelationKeyInfo_name on CorrelationKeyInfo(name);
```
Comment (gmunozfe) [inline jbpm-db-scripts/src/main/resources/db/upgrade-scripts/oracle/rhpam-7.11-to-7.12-CorrelationKeyInfo_name.sql:1]: Same

File: jbpm-db-scripts/src/main/resources/db/ddl-scripts/oracle/oracle-jbpm-schema.sql [hunk 19]
```diff
@@ -893,4 +893,13 @@
 
     create index IDX_TaskVariableImpl_taskId on TaskVariableImpl(taskId);
     create index IDX_TaskVariableImpl_pInstId on TaskVariableImpl(processInstanceId);
-    create index IDX_TaskVariableImpl_processId on TaskVariableImpl(processId);
\ No newline at end of file
+    create index IDX_TaskVariableImpl_processId on TaskVariableImpl(processId);
+    
+    
+    create unique index IDX_CorrelationKeyInfo_name on CorrelationKeyInfo(name);
```
Comment (gmunozfe) [inline jbpm-db-scripts/src/main/resources/db/ddl-scripts/oracle/oracle-jbpm-schema.sql:899]: Same

File: jbpm-db-scripts/src/main/resources/db/ddl-scripts/sybase/sybase-jbpm-schema.sql [hunk 20]
```diff
@@ -1021,4 +1021,14 @@
     create index IDX_TaskVariableImpl_pInstId on TaskVariableImpl(processInstanceId)
     go
     create index IDX_TaskVariableImpl_processId on TaskVariableImpl(processId)
+    go
+    create unique index IDX_CorrelationKeyInfo_name on CorrelationKeyInfo(name)
```
Comment (gmunozfe) [inline jbpm-db-scripts/src/main/resources/db/ddl-scripts/sybase/sybase-jbpm-schema.sql:1025]: Same, though it doesn't fail even if already exists

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/sybase/jbpm-7.58-to-7.59.sql [hunk 21]
```diff
@@ -0,0 +1,10 @@
+create unique index IDX_CorrelationKeyInfo_name on CorrelationKeyInfo(name)
+go
```
Comment (gmunozfe) [inline jbpm-db-scripts/src/main/resources/db/upgrade-scripts/sybase/jbpm-7.58-to-7.59.sql:2]: Same

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/sybase/rhpam-7.11-to-7.12.sql [hunk 22]
```diff
@@ -10,4 +10,14 @@ alter table VariableInstanceLog add column end_date datetime
 go
 create index IDX_TaskEvent_processInstanceId on TaskEvent (processInstanceId)
 go
+create unique index IDX_CorrelationKeyInfo_name on CorrelationKeyInfo(name)
+go
```
Comment (gmunozfe) [inline jbpm-db-scripts/src/main/resources/db/upgrade-scripts/sybase/rhpam-7.11-to-7.12.sql:14]: Same

File: jbpm-db-scripts/src/main/resources/db/ddl-scripts/db2/db2-jbpm-schema.sql [hunk 23]
```diff
@@ -824,4 +824,14 @@
 
     create index IDX_TaskVariableImpl_taskId on TaskVariableImpl(taskId);
     create index IDX_TaskVariableImpl_pInstId on TaskVariableImpl(processInstanceId);
-    create index IDX_TaskVariableImpl_processId on TaskVariableImpl(processId);
\ No newline at end of file
+    create index IDX_TaskVariableImpl_processId on TaskVariableImpl(processId);
+
+    create unique index IDX_CorrelationKeyInfo_name on CorrelationKeyInfo(name);
```
Comment (gmunozfe) [inline jbpm-db-scripts/src/main/resources/db/ddl-scripts/db2/db2-jbpm-schema.sql:829]: Same

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/db2/jbpm-7.58-to-7.59.sql [hunk 24]
```diff
@@ -0,0 +1,5 @@
+create unique index IDX_CorrelationKeyInfo_name on CorrelationKeyInfo(name);
```
Comment (gmunozfe) [inline jbpm-db-scripts/src/main/resources/db/upgrade-scripts/db2/jbpm-7.58-to-7.59.sql:1]: Same

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/db2/rhpam-7.11-to-7.12.sql [hunk 25]
```diff
@@ -4,5 +4,10 @@ alter table TaskEvent add column end_date timestamp;
 alter table NodeInstanceLog add column end_date timestamp;
 alter table VariableInstanceLog add column end_date timestamp;
 create index IDX_TaskEvent_processInstanceId on TaskEvent (processInstanceId);
+create unique index IDX_CorrelationKeyInfo_name on CorrelationKeyInfo(name);
```
Comment (gmunozfe) [inline jbpm-db-scripts/src/main/resources/db/upgrade-scripts/db2/rhpam-7.11-to-7.12.sql:7]: Same

**Review by gmunozfe — CHANGES_REQUESTED** [review]
_IDX_CorrelationKeyInfo_name_ already exists in scripts, as 
alter table CorrelationKeyInfo add constraint IDX_CorrelationKeyInfo_name unique (name);
was introduced with https://github.com/kiegroup/jbpm/commit/ef55b48ac429ba13f97b5ddc9e5a3ec81f4f3146

So, there's no need to create it as unique index

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/postgresql/jbpm-7.58-to-7.59.sql (hunk 5 above)
Comment (elguardian) [reply jbpm-db-scripts/src/main/resources/db/upgrade-scripts/postgresql/jbpm-7.58-to-7.59.sql:1]: done

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/postgresql/rhpam-7.11-to-7.12.sql (hunk 6 above)
Comment (elguardian) [reply jbpm-db-scripts/src/main/resources/db/upgrade-scripts/postgresql/rhpam-7.11-to-7.12.sql:7]: done

File: jbpm-db-scripts/src/main/resources/db/ddl-scripts/postgresql/postgresql-jbpm-schema.sql [hunk 26]
```diff
@@ -894,4 +894,13 @@
 
     create index IDX_TaskVariableImpl_taskId on TaskVariableImpl(taskId);
     create index IDX_TaskVariableImpl_pInstId on TaskVariableImpl(processInstanceId);
-    create index IDX_TaskVariableImpl_processId on TaskVariableImpl(processId);
\ No newline at end of file
+    create index IDX_TaskVariableImpl_processId on TaskVariableImpl(processId);
+    
+    create unique index IDX_CorrelationKeyInfo_name on CorrelationKeyInfo(name);
```
[the remark this answers is not in the document]
Comment (elguardian) [reply jbpm-db-scripts/src/main/resources/db/ddl-scripts/postgresql/postgresql-jbpm-schema.sql:899]: done

File: jbpm-db-scripts/src/main/resources/db/ddl-scripts/sqlserver2008/sqlserver2008-jbpm-schema.sql (hunk 4 above)
Comment (elguardian) [reply jbpm-db-scripts/src/main/resources/db/ddl-scripts/sqlserver2008/sqlserver2008-jbpm-schema.sql:833]: done

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/sqlserver/jbpm-7.58-to-7.59.sql (hunk 7 above)
Comment (elguardian) [reply jbpm-db-scripts/src/main/resources/db/upgrade-scripts/sqlserver/jbpm-7.58-to-7.59.sql:1]: done

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/sqlserver/rhpam-7.11-to-7.12.sql (hunk 8 above)
Comment (elguardian) [reply jbpm-db-scripts/src/main/resources/db/upgrade-scripts/sqlserver/rhpam-7.11-to-7.12.sql]: done

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/sqlserver2008/jbpm-7.58-to-7.59.sql (hunk 9 above)
Comment (elguardian) [reply jbpm-db-scripts/src/main/resources/db/upgrade-scripts/sqlserver2008/jbpm-7.58-to-7.59.sql]: done

File: jbpm-db-scripts/src/main/resources/db/ddl-scripts/mysql5/mysql5-jbpm-schema.sql (hunk 11 above)
Comment (elguardian) [reply jbpm-db-scripts/src/main/resources/db/ddl-scripts/mysql5/mysql5-jbpm-schema.sql]: done

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/mysql5/jbpm-7.58-to-7.59.sql (hunk 13 above)
Comment (elguardian) [reply jbpm-db-scripts/src/main/resources/db/upgrade-scripts/mysql5/jbpm-7.58-to-7.59.sql]: done

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/mysqlinnodb/jbpm-7.58-to-7.59.sql (hunk 15 above)
Comment (elguardian) [reply jbpm-db-scripts/src/main/resources/db/upgrade-scripts/mysqlinnodb/jbpm-7.58-to-7.59.sql]: done

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/mysqlinnodb/rhpam-7.11-to-7.12.sql (hunk 16 above)
Comment (elguardian) [reply jbpm-db-scripts/src/main/resources/db/upgrade-scripts/mysqlinnodb/rhpam-7.11-to-7.12.sql]: done

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/oracle/jbpm-7.58-to-7.59-CorrelationKeyInfo_name.sql (hunk 17 above)
Comment (elguardian) [reply jbpm-db-scripts/src/main/resources/db/upgrade-scripts/oracle/jbpm-7.58-to-7.59-CorrelationKeyInfo_name.sql]: done

File: jbpm-db-scripts/src/main/resources/db/ddl-scripts/oracle/oracle-jbpm-schema.sql (hunk 19 above)
Comment (elguardian) [reply jbpm-db-scripts/src/main/resources/db/ddl-scripts/oracle/oracle-jbpm-schema.sql]: done

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/oracle/rhpam-7.11-to-7.12-CorrelationKeyInfo_name.sql (hunk 18 above)
Comment (elguardian) [reply jbpm-db-scripts/src/main/resources/db/upgrade-scripts/oracle/rhpam-7.11-to-7.12-CorrelationKeyInfo_name.sql]: done

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/sybase/rhpam-7.11-to-7.12.sql (hunk 22 above)
Comment (elguardian) [reply jbpm-db-scripts/src/main/resources/db/upgrade-scripts/sybase/rhpam-7.11-to-7.12.sql]: done

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/sybase/jbpm-7.58-to-7.59.sql (hunk 21 above)
Comment (elguardian) [reply jbpm-db-scripts/src/main/resources/db/upgrade-scripts/sybase/jbpm-7.58-to-7.59.sql]: done

File: jbpm-db-scripts/src/main/resources/db/ddl-scripts/db2/db2-jbpm-schema.sql (hunk 23 above)
Comment (elguardian) [reply jbpm-db-scripts/src/main/resources/db/ddl-scripts/db2/db2-jbpm-schema.sql]: done

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/db2/rhpam-7.11-to-7.12.sql (hunk 25 above)
Comment (elguardian) [reply jbpm-db-scripts/src/main/resources/db/upgrade-scripts/db2/rhpam-7.11-to-7.12.sql]: done

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/db2/jbpm-7.58-to-7.59.sql (hunk 24 above)
Comment (elguardian) [reply jbpm-db-scripts/src/main/resources/db/upgrade-scripts/db2/jbpm-7.58-to-7.59.sql]: done

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/mysql5/rhpam-7.11-to-7.12.sql (hunk 14 above)
Comment (elguardian) [reply jbpm-db-scripts/src/main/resources/db/upgrade-scripts/mysql5/rhpam-7.11-to-7.12.sql]: done

File: jbpm-db-scripts/src/main/resources/db/ddl-scripts/mysqlinnodb/mysql-innodb-jbpm-schema.sql (hunk 12 above)
Comment (elguardian) [reply jbpm-db-scripts/src/main/resources/db/ddl-scripts/mysqlinnodb/mysql-innodb-jbpm-schema.sql]: done

File: jbpm-db-scripts/src/main/resources/db/upgrade-scripts/sqlserver2008/rhpam-7.11-to-7.12.sql (hunk 10 above)
Comment (elguardian) [reply jbpm-db-scripts/src/main/resources/db/upgrade-scripts/sqlserver2008/rhpam-7.11-to-7.12.sql]: done

File: jbpm-db-scripts/src/main/resources/db/ddl-scripts/sybase/sybase-jbpm-schema.sql (hunk 20 above)
Comment (gmunozfe) [inline jbpm-db-scripts/src/main/resources/db/ddl-scripts/sybase/sybase-jbpm-schema.sql:1025]: still present

**Review by gmunozfe — APPROVED** [review]
Just fix sybase, all the rest worked fine, great job @elguardian !

**Pull request merged**